tests/ClientTest.php

```diff
@@ -144,22 +146,22 @@ public function testCaptureEvent(): void
             ->setTransportFactory($this->createTransportFactory($transport))
             ->getClient();
 
-        $inputData = [
-            'transaction' => 'foo bar',
-            'level' => Severity::debug(),
-            'logger' => 'foo',
-            'tags_context' => ['foo', 'bar'],
-            'extra_context' => ['foo' => 'bar'],
-            'user_context' => ['bar' => 'foo'],
-        ];
+        $event = Event::createEvent();
+
```

Comment:
You can remove this blank line as the statements below are related to the preceding line